rules/react/off.js

```diff
@@ -32,23 +36,23 @@ module.exports = {
     // Enforce ES5 or ES6 class for React Components
     "react/prefer-es6-class": 0,
     // Enforce stateless React Components to be written as a pure function
-    "prefer-stateless-function": 0,
+    "react/prefer-stateless-function": 0,
```

Comment:
The only annoying thing about this is that hot loaders still don't support stateless functional components AFAIK (anyone know if that's still true?) – sometimes I turn components back into classes just because I really want hot reload to work during development.